packages/amplify_auth_cognito/test/amplify_auth_cognito_confirmSignIn_test.dart

```diff
@@ -19,6 +19,23 @@ import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:amplify_auth_cognito/method_channel_auth_cognito.dart';
 import 'package:amplify_core/amplify_core.dart';
 
+extension IsEqual on SignInResult {
+  bool isMostlyEqual(SignInResult comparator) {
```

Comment:
Which is/are the field(s) that are being ignored? Can you annotate this somewhere?

Reply:
No fields are being ignored - the additionalInfo attribute is just a map of key values, so it could take a bit of work to recursively check for equality within the potential maps it may contain.  For the purposes of this test, I think checking the length is adequate. I left a comment.